integration-tests/cli/discount-codes.it.js

```diff
@@ -308,31 +265,27 @@ describe('DiscountCode tests', () => {
         maxApplications: 10,
         maxApplicationsPerCustomer: 2,
       }
+      try {
+        const [stdout, stderr] = await exec(
+          `${bin} -o ${jsonFilePath} -p ${projectKey}`
+        )
+        expect(stderr).toBeFalsy()
+        expect(stdout).toMatchSnapshot()
 
-      exec(
-        `${bin} -o ${jsonFilePath} -p ${projectKey}`,
-        (cliError, stdout, stderr) => {
-          expect(cliError).toBeFalsy()
-          expect(stderr).toBeFalsy()
-          expect(stdout).toMatch(/Export operation completed successfully/)
-
-          fs.readFile(jsonFilePath, { encoding: 'utf8' }, (error, data) => {
-            expect(error).toBeFalsy()
-            const actual = JSON.parse(data)
-            actual.forEach(codeObj => {
-              expect(codeObj).toMatchObject(expected)
-              expect(isuuid(codeObj.id)).toBe(true)
-              expect(codeObj.createdAt).toMatch(UTCDateTimeRegex)
-              expect(codeObj.lastModifiedAt).toMatch(UTCDateTimeRegex)
-            })
-
-            done()
-          })
-        }
-      )
+        data = await fs.readFile(jsonFilePath, { encoding: 'utf8' })
+      } catch (error) {
```

Comment:
This catch block should be removed. As we don't expect any errors, a rejected promise will fail the test.

Reply:
Good one, fixed!